test/unit/specs/App.spec.js

```diff
@@ -0,0 +1,113 @@
+import App from '@/App'
```

Comment:
Fixed in https://github.com/JakubPetriska/timezones/pull/1/commits/d47add8953d8eae4902b1b5457fd10f22757ce29 .